src/main/java/by/byshnev/controllers/ArticlesRestController.java

```diff
@@ -0,0 +1,24 @@
+package by.byshnev.controllers;
+
+import by.byshnev.dto.ArticleDto;
+import by.byshnev.services.ArticleServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/articles")
+public class ArticlesRestController {
+
+    private final ArticleServiceImpl articleServiceImpl;
```

Comment:
always inject **interfaces** instead of particular classes. controller doesn't care what actual service implementation it uses.